spec/lib/submission_builder/ty2024/states/nj/nj_return_xml_spec.rb

```diff
@@ -61,22 +65,28 @@
         it "does not error" do
           builder_response = described_class.build(submission)
           expect(builder_response.errors).not_to be_present
+          expect(builder_response.document.at("WagesSalariesTips").text).not_to eq(nil)
+          expect(builder_response.document.at("NewJerseyTaxableIncome").text).not_to eq(nil)
         end
       end
 
       context "with many w2s" do
-        let(:intake) { create(:state_file_nj_intake, municipality_code: "0101", raw_direct_file_data: StateFile::DirectFileApiResponseSampleService.new.read_xml('nj_zeus_many_w2s')) }
+        let(:intake) { create(:state_file_nj_intake, raw_direct_file_data: StateFile::DirectFileApiResponseSampleService.new.read_xml('nj_zeus_many_w2s')) }
```

Comment:
[pebble] Can we remove the raw_direct_file_data here and just use :df_data_many_w2s?